mappings/net/minecraft/network/listener/C2SPacketListener.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_lpmetgbe net/minecraft/network/listener/C2SPacketListener
```

Comment:
This name is contradictory to existing names:
A matching class for `S2C` is named `ClientPacketListener`, solving the "we already have that just below it in the inheritance tree" problem by renaming the other class `ClientPacketListener` -> `ClientCommonPacketListener`. I'm open to criticism on this decision (it was mine) but I think this is the better choice.